plugin/restore/mysql/mysql.go

```diff
@@ -520,9 +531,60 @@ func binlogFilesAreContinuous(files []BinlogFile) bool {
 	return true
 }
 
-// FetchArchivedBinlogFiles downloads the missing binlog files from the remote instance to `binlogDir`,
-// but exclude latest binlog. We may  download the latest binlog only when doing PITR.
-func (r *Restore) FetchArchivedBinlogFiles(ctx context.Context) error {
+// Re-download inconsistent local binlog files.
+// This is a best-effort task, because the corresponding binlog files may not exist on server.
```

Comment:
Any reason we need to do it separately? Can we just use a single function to download all binary logs on the server that's not local? "Not local" means not exists or exists with inconsistent size.

Reply:
Refactored.